src/tools/dashboard/src/books.rs

```diff
@@ -283,7 +283,10 @@ fn prepend_props(path: &Path, example: &mut Example, config_paths: &mut HashSet<
 
 /// Make the main function of a test public so it can be verified by rmc.
 fn pub_main(code: String) -> String {
-    code.replace("fn main", "pub fn main")
```

Comment:
I don't think we need this hack anymore. We should be able to verify binary crates now.

Reply:
https://github.com/model-checking/rmc/pull/656

Reply:
Thanks!! I have left a review there and hope we can merge it soon. Should we reuse this PR for the warning changes or close it and open a new one with them?

Reply:
We can reuse it. Maybe just make sure you edit the title!

Reply:
Updated PR to contain only the warnings changes.